.gitignore

```diff
@@ -33,6 +33,6 @@ JWT_TESTING_WITH_ASID_PRIVATE_KEY
 keys/
 
 
-target/
-karate-tests/target/
-karate-tests/src/test/java/target/
+# Karate ignores
+**/target/**
```

Comment:
I think this maybe too wide a criteria, better to be a bit more specific

Reply:
In a java project, the folder created after compilation is "target". If there's an issue with this rule - if at some point we decide to add a folder called "target" that doesn't relate to Java compilation (unlikely to happen and arguably not the best name given Java conventions) - then we can always review but for now I think it's preferable to have this wide net rather than list everywhere a "target" folder might be created (the target folder is created relative to where you run commands).

Reply:
okay, that's reasonable - you've answered the potential user case - as long as it won't cause a problem and we remember it's in gitignore.